```python
a = -1
# 断言 a>0才正常执行，否则抛 异常
assert a>0

```
[Out]:

```
Traceback (most recent call last):
  File "D:\Anaconda3\envs\tensorflow\lib\site-packages\IPython\core\interactiveshell.py", line 3331, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-45-013be11a8622>", line 3, in <module>
    assert a>0
AssertionError
```
